lib/ui/Select/select.tsx

```diff
@@ -0,0 +1,132 @@
+"use client"
```

Comment:
We don't really need this, right?

Reply:
yes, i know, but i included this PR https://github.com/factorialco/factorial-one/pull/1313 to be able to test the select in the monorepo without need to merge this branch to main